sys_tests/snmp_proxy.py

```diff
@@ -0,0 +1,55 @@
+from os import path
+import time
+
+from influxdb import InfluxDBClient
+
+from cosmo_tester.framework.testenv import TestCase
+
+
+class TestSNMPProxy(TestCase):
+    TIME_TO_WAIT_FOR_METRICS = 10  # in seconds
+
+    def _test_snmp_monitoring(self, blueprint, inputs):
+        self.blueprint_yaml = path.join(
+            path.dirname(path.dirname(path.realpath(__file__))),
+            blueprint
+        )
+        deployment_id = self.test_id
+        self.upload_deploy_and_execute_install(
+            inputs=inputs,
+            deployment_id=deployment_id
+        )
+
+        # Check that proper metrics are stored
+        time.sleep(self.TIME_TO_WAIT_FOR_METRICS)
```

Comment:
Please change the sleep to something more robust by using something like here
https://github.com/cloudify-cosmo/cloudify-system-tests/blob/master/cosmo_tester/framework/testenv.py#L397
so for instance, instead of waiting 10 seconds and the asserting, wait within a loop, the checks every second and times out after 30 seconds